sklearn/feature_selection/_mutual_info.py

```diff
@@ -306,6 +308,14 @@ def _estimate_mi(
         for x, discrete_feature in zip(_iterate_columns(X), discrete_mask)
     ]
 
+    def _compute_mi_single_var(x, discrete_feature):
```

Comment:
I don't think that we need this function: we can just pass the parameter in the function below.